### PR TITLE
fix: include tax note in wishlist

### PR DESCRIPTION
**Pull request by rbedeoan-plenty**
[AB#131913](https://dev.azure.com/plentymarkets/0af57df9-8662-4901-bb97-8e88b07ff0c9/_workitems/edit/131913)

### CODE REVIEW

File: apps/web/components/WishlistPageContent/WishlistPageContent.vue [hunk 1]
```diff
@@ -71,6 +77,8 @@ import { productGetters, productImageGetters } from '@plentymarkets/shop-api';
 import { SfLoaderCircular } from '@storefront-ui/vue';
 import type { WishlistPageContentProps } from '~/components/WishlistPageContent/types';
 
+const runtimeConfig = useRuntimeConfig();
+const showNetPrices = runtimeConfig.public.showNetPrices;
```
Comment (stentrop) [inline apps/web/components/WishlistPageContent/WishlistPageContent.vue:81]: im not sure if this is a temporarily solution but in general the showNetPrice should be dynamic

**Review by stentrop — CHANGES_REQUESTED** [review]
check my single comment

File: apps/web/components/WishlistPageContent/WishlistPageContent.vue (hunk 1 above)
Comment (abocsan-plenty) [reply apps/web/components/WishlistPageContent/WishlistPageContent.vue:81]: This should be temporary only, I created another PBI to properly fix this:

https://dev.azure.com/plentymarkets/plentymarkets/_workitems/edit/132247

**Review by abocsan-plenty — APPROVED** [review]

File: apps/web/components/WishlistPageContent/WishlistPageContent.vue (hunk 1 above)
Comment (rbedeoan-plenty) [reply apps/web/components/WishlistPageContent/WishlistPageContent.vue:81]: thank you for the feedback, for now, I'll proceed with this temporary solution as discussed with abocsan-plenty

**Review by stentrop — APPROVED** [review]

**Pull request merged**